composer.json

```diff
@@ -16,12 +16,12 @@
     "psr/log": "^1.0",
     "symfony/symfony": "^2.6|^3.0",
     "doctrine/orm": "^2.3",
-    "treehouselabs/queue": "^v0.1.0"
+    "treehouselabs/queue": "dev-master"
```

Comment:
This will be updated to `^v0.2.0` when I've functionally tested this in an actual environment, and the queue lib gets a new tag.